codeworld-api/src/CodeWorld/Color.hs

```diff
@@ -93,6 +97,30 @@ gray, grey :: Double -> Color
 gray = grey
 grey k = RGBA k k k 1
 
+
+-- Produce an infinite list of colors:
+-- https://github.com/google/codeworld/issues/436
```

Comment:
I'm generally opposed to leaving breadcrumbs in comments.  Historical information like this belongs in commit logs rather than cluttering the source code.  Instead, just write a one-line description of the purpose in the exported function (`colors`)?  Better if it's a haddock comment, especially since this one is non-obvious.